helm/dagster/templates/serviceaccount.yaml

```
@@ -5,4 +5,6 @@ metadata:
   name: {{ include "dagster.serviceAccountName" . }}
   labels:
 {{ include "dagster.labels" . | nindent 4 }}
-{{- end -}}
\ No newline at end of file
+  annotations:
+{{- .Values.serviceAccount.annotations | toYaml | nindent 4 }}
```

nit: let's collapse this into one line so there's no ambiguity about the spacing. `nindent` already puts this block on the new line anyways.
```suggestion
  annotations: {{ .Values.serviceAccount.annotations | toYaml | nindent 4 }}
```

---

Roger. I agree - I placed it on the next line to mirror `labels` on the line above. Maybe they should both be inlined?